WORKSPACE

```diff
@@ -36,7 +36,14 @@ protobuf_deps()
 git_repository(
     name = "com_google_absl",
     remote = "https://github.com/abseil/abseil-cpp.git",
-    tag = "20211102.0",
+    tag = "20210324.2",
```

Comment:
This syncs abseil-cpp to an earlier version than we currently do.  I would prefer not go backwards in time, unless there is a reason.

Reply:
Ok, I will modify to 20211102.0 version.